test/compress/properties.js

```diff
@@ -2486,3 +2486,15 @@ mangle_undeclared_properties: {
         var r={o:function(){return a.t()}};
     }
 }
+
+keep_numbers_in_properties_as_is: {
+    beautify = {
+        keep_numbers: true
+    }
+    input: {
+        var Foo = { 1000000: 80000000000 }
+    }
+    expect: {
+        var Foo = { 1000000: 80000000000 }
```

Comment:
This needs to be `expect_exact`, otherwise it leads to false positives.

Reply:
Thanks, fixed!